static/js/blame.js

```diff
@@ -180,8 +200,10 @@ var BlameStripHoverHandler = new (class BlameStripHoverHandler {
     // pop-up.
     if (
       event.type == "mouseleave" ||
-      (event.type == "click" && BlamePopup.blameElement)
+      event.type == "scroll" ||
+      (event.type == "click" && clickedOutsideBlameStrip)
```

Comment:
instead of `(event.type == "click" && clickedOutsideBlameStrip)` just `clickedOutsideBlameStrip`.